components/learn/StartLearningSection.vue

```diff
@@ -342,3 +352,24 @@ export default class StartLearningSection extends Vue {
   }
 }
 </style>
+
+<style lang="scss">
+  .start-learning-section {
+      // override tab position
+    .cv-tab.bx--tabs {
+      display: flex;
+      justify-content: flex-end;
+      padding-bottom: 3rem;
```

Comment:
Another thing I see happening, is that at smaller screens, the tabs turns into a dropdown — the dropdown component _should_ be full width, at smaller breakpoints.

![Screen Shot 2022-10-10 at 3 02 07 PM](https://user-images.githubusercontent.com/6276074/194959222-40b7a199-bce5-4478-858f-c79fc8d77d31.png)


We can fix that by updating the styles like so:

```css
    @include mq($until: medium) {
        display: block;
    }
```


(we can review this during our sync as well)